docs/source/tutorial.rst

```diff
@@ -182,44 +244,225 @@ Sawtooth Lake includes additional transaction processors:
+[19:31:06 INFO    handler] processing: Verb=set Name=lrnuDC Value=92318 address=1cf12617c797cf8c27254bbdb5c9bda09f9405b9494ae32b79b9b6d30881ca8552d5932a68f703d1b6754b9feb2edafa76a797fc0826110381b0f8614f2c6853316b47
+[19:31:06 INFO    handler] processing: Verb=set Name=BKaiql Value=94175 address=1cf12669cbc17d076a1accb4b0bb61f40ed4f999173b90e3ca2591875a55fee2947661e60fa1c57b41ef0f2660176b945a01c85ff645543297068a3fb1306324a19612
+[19:31:06 INFO    handler] processing: Verb=set Name=wpMQmE Value=47316 address=1cf1260f6bdf66b65ff7c00ec58c4deccffd167bfee7a85698880dfa485df3de1ec18a5b2d1dc12849743d1c74320108360a2d40d223b35fbc1c4ea03bbd8306480c62
+[19:31:06 INFO    handler] processing: Verb=set Name=GTgrvP Value=31921 address=1cf12606ac7db03c756133c07d7d02b59f3ef9eae6774fe59c75c88ab66a9fabbbaef9975dbf9aa197d1090ed126d7b18e2
+
+
+Config Transaction Family Usage
+===============================
 
-Enter the following series of commands after logging in to the development 
-environment with 'vagrant ssh':
+Sawtooth Lake provides a :doc:`config transaction family
+<transaction_family_specifications/config_transaction_family>` that stores on-
+chain configuration settings, along with a config family transaction
+processor written in Python.
+
+One of the on-chain settings is the list of supported transaction families. 
+To configure this setting, follow these steps:
+
+Step One: Start Config Family Processor
+---------------------------------------
+
+To start the config family transaction processor, run the following commands from the 
+Vagrant CLI:
+
+.. code-block:: console
+
+  $ tp_config tcp://localhost:40000
+
+Confirm that the transaction processor registers with the validator by viewing the Vagrant shell 
+in which the validator is running. A successful registration event produces the following output:
+
+.. code-block:: console
+  
+  [21:03:55.955 INFO    processor_handlers] registered transaction processor: identity=b'6d2d80275ae280ea', family=sawtooth_config, version=1.0, encoding=application/protobuf, namespaces=<google.protobuf.pyext._message.RepeatedScalarContainer object at 0x7e1ff042f6c0>
+  [21:03:55.956 DEBUG   interconnect] ServerThread sending TP_REGISTER_RESPONSE to b'6d2d80275ae280ea'
+
+
+Step Two: Create And Submit Batch
+----------------------------------
+
+In the example below, a JSON array is submitted to the `sawtooth config`
+command, which creates a batch of transactions containing the configuration
+change. The `intkey load` command is then used to submit the batch to the
+validator.
+
+The JSON array used tells the validator or validator network to accept transactions of the following types:
+
+* intkey
+* sawtooth_config
+
+To create and submit the batch containing the new configuration, enter the
+following commands from the Vagrant CLI:
+
+.. code-block:: console
+
+  $ sawtooth admin keygen my_key
```

Comment:
For this step, the user can actually run regular `sawtooth keygen` and use the key generated from that process.